src/libs/wrapPageElement.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react";
+import { ThemeProvider, CssBaseline, Box } from "@mui/material";
+import "@fontsource/poppins";
+import { customTheme } from "../themes/theme";
+
+const wrapPageElement = ({ element }: { element: ReactNode }) => {
```

Comment:
Document why we need it

Reply:
done

Reply:
@Ayomikuolatunji where did you documented the purpose of this file? I still see the code below. Write the comment in this file.
```
import React, { ReactNode } from "react";
import { ThemeProvider, CssBaseline, Box } from "@mui/material";
import "@fontsource/poppins";
import { customTheme } from "../themes/theme";

const wrapPageElement = ({ element }: { element: ReactNode }) => {
  return (
    <ThemeProvider theme={customTheme}>
      <CssBaseline />
      <Box>{element}</Box>
    </ThemeProvider>
  );
};

export default wrapPageElement;

```